dataeng/jobs/analytics/RetirementJobs.groovy

```diff
@@ -59,15 +59,15 @@ class RetirementJobs{
             wrappers common_wrappers(allVars)
             parameters secure_scm_parameters(allVars)
             parameters {
-                stringParam('PLATFORM_BRANCH', 'origin/2u/release', 'Branch from the edx-platform repository. For tags use tags/[tag-name].')
+                stringParam('TUBULAR_BRANCH', 'master', 'Repo branch for the tubular scripts.')
                 stringParam('ENVIRONMENT', '', 'edx environment which contains the user in question, in ENVIRONMENT-DEPLOYMENT format.')
                 stringParam('RETIREMENT_USERNAME', '', 'Current username of learner to retire.')
             }
 
             // retry cloning repositories
             checkoutRetryCount(5)
 
-            multiscm platform_scm(allVars) << {
+            multiscm (allVars) << {
```

Comment:
I think we need to remove (allVars):
` multiscm (allVars) << {` -> ` multiscm  {`